src/codegen/runtime_stubs.rs

```diff
@@ -72,9 +72,11 @@ pub const I64_REM: &str = "i64_rem";
 
 pub const F32_MIN: &str = "f32_min";
 pub const F32_MAX: &str = "f32_max";
+pub const F32_FLOOR: &str = "f32_floor";
 
 pub const F64_MIN: &str = "f64_min";
 pub const F64_MAX: &str = "f64_max";
+pub const F64_FLOOR: &str = "f64_floor";
```

Comment:
I'm a huge nitpicker, but can you alphabetize these blocks?

Reply:
DONE!